react/components/admin/institutional/Form/withContentContext.tsx

```diff
@@ -103,9 +106,44 @@ function withContentContext<T>(
                   return <Loader />
                 }
 
-                const blockId = dataTemplates!.availableTemplates.filter(
+                const elegibleTemplates = dataTemplates!.availableTemplates.filter(
                   (template: Template) => template.id !== interfaceId
-                )[0].id
+                )
+
+                if (!elegibleTemplates.length) {
+                  return (
+                    <div>
+                      <EmptyState
+                        title={
+                          <FormattedMessage
+                            id="admin/pages.admin.content.not-supported.title"
+                            defaultMessage="This feature is not available :/"
+                          />
+                        }
+                      >
+                        <p>
+                          <FormattedMessage id="admin/pages.admin.content.not-supported.description" />
+                        </p>
+                        <div className="pt5">
+                          <Button variation="secondary" size="small">
+                            <a
```

Comment:
The `Button` component already has props for links, such as `href`, `target`, etc.
(https://styleguide.vtex.com/#/Components/Forms/Button)

Reply:
👍